openvdb/openvdb/tree/RootNode.h

```diff
@@ -2388,9 +2388,11 @@ RootNode<ChildT>::readTopology(std::istream& is, bool fromHalf)
     // Delete the existing tree.
     this->clear();
 
-    // Read a RootNode that was stored in the current format.
+    // converting reader for reading grid values
+    auto& convertingReader = io::ConvertingReader<ValueType>::get(is);
```

Comment:
What makes more sense is to move this to `readTopologyWithValueType` since this function is templated with `SourceValueT`.